application/common/components/exceptions/MissingEmailException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace Sil\Idp\IdSync\common\components\exceptions;
+
+use Exception;

```

Comment:
Should this be `use \Exception;` since there is no `Sil\Idp\IdSync\common\components\exceptions\Exception` class?